src/lib/viewers/controls/annotations/AnnotationsButton.tsx

```diff
@@ -13,21 +13,17 @@ export type Props = Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'onClick
     onClick?: (mode: AnnotationMode) => void;
 };
 
-export default function AnnotationsButton({
-    children,
-    className,
-    isActive = false,
-    isEnabled = true,
-    mode,
-    onClick = noop,
-    ...rest
-}: Props): JSX.Element | null {
+function AnnotationsButton(
+    { children, className, isActive = false, isEnabled = true, mode, onClick = noop, ...rest }: Props,
```

Comment:
Super minor, but we generally use this function signature for forwarding refs, since it can get awkward:

```
function AnnotationsButton(props: Props, ref: React.Ref<HTMLButtonElement>) {
    const { children, className, isActive = false, isEnabled = true, mode, onClick = noop, ...rest } = props;
    // ...etc
}